source/shared/net/desktop-wallet/accounts-desktop.rst

```diff
@@ -25,7 +25,16 @@ The most common tasks on an account are:
 
 In addition, you can see the following information about the selected account:
 
--  **Account Total**: The total number of GTU on the account, which includes the GTU that is available and any staked GTU or shielded GTU. It also includes GTU that is scheduled to be released to your account on a future date. The **Balance** shows the GTU that are at your disposal, the amount of GTU that you've staked, and any GTU that are locked in a release schedule. The **At disposal** amount is the amount you can use for transactions, which excludes staked GTU and GTU in a release schedule. The **Shielded balance** shows the amount of GTU that you've shielded. The **Balance** is visible to all participants on the Concordium Blockchain, while the **Shielded balance** is visible to you only.
+-  The **Account Total** shows the total number of GTU on the account, which includes the GTU that is available and any staked GTU or shielded GTU. It also includes GTU that is scheduled to be released to your account on a future date.
+
+-  The **Balance** shows the GTU that are at your disposal, the amount of GTU that you've staked, and any GTU that are locked in a release schedule.
+
+-  The **At disposal** amount is the amount you can use for transactions, which excludes staked GTU and GTU in a release schedule.
```

Comment:
Should we mention that it also excludes the shielded balance, or is that clear enough already?